index.html

```diff
@@ -237,4 +238,4 @@ <h2 class="Contact">Contact me</h2>
     </section>
```

Comment:
- [ ] Kindly as you center the form consider sticking to the Figma design by making sure that the `get in touch` button is also centered and also giving your paragraph a small width to make your project an exact duplicate of the provided design
- your design 👇 
<img width="764" alt="Screen Shot 2023-05-12 at 23 42 34" src="https://github.com/MarkYegon7/Portfolio-Setup/assets/60146030/a238f1c3-2b02-4b9a-b2d1-f1fac7eb38b1">

- Figma design 👇 
<img width="741" alt="Screen Shot 2023-05-12 at 23 42 01" src="https://github.com/MarkYegon7/Portfolio-Setup/assets/60146030/d4509373-d383-4cab-a9fe-3049f3deb1c2">

Reply:
Made the required changes